README.md

```diff
@@ -2444,23 +2452,23 @@ ipmitool -I bmc lan set 1 defgw ipaddr 192.168.0.1
 
 ##### Resolve a domain to IP address(es)
 ```bash
-dig +short www.example.com
+dig +short www.example.net
 
 # or
-host www.example.com
+host www.example.net
 ```
 
 ##### Get DNS TXT record a of domain
 ```bash
-dig -t txt www.example.com
+dig -t txt www.example.net
 
 # or
-host -t txt www.example.com
+host -t txt www.example.net
 ```
 
 ##### Send a ping with a limited TTL to 10 (TTL: Time-To-Live, which is the maximum number of hops that a packet can travel across the Internet before it gets discarded.)
 ```bash
-ping 8.8.8.8 -t 10
+ping 192.0.2.1 -t 10
```

Comment:
I don't this packet would travel across the internet but it would with 8.8.8.8

Reply:
by using the RFC:5737 `TEST-NET-1` network address `192.0.2.1` you are not promoting any trackers/spyware or any other malicious address. By mention any particular DNS recursor, would only end up in a religious war or mistrust on the idea for this cheat sheet. This is why I changed the bad resolver to a "fictive" address for neutrality.